src/plugins/condition/ConditionManager.js

```diff
@@ -226,11 +226,12 @@ export default class ConditionManager extends EventEmitter {
         }
 
         this.openmct.objects.get(currentConditionIdentifier).then((obj) => {
-            this.emit('conditionSetResultUpdated', {
-                id: this.domainObject.identifier,
-                output: obj.configuration.output,
-                conditionId: currentConditionIdentifier
-            })
+            this.emit('conditionSetResultUpdated',
+                Object.assign({},
+                    resultObj ? resultObj.data : {},
+                    { output: obj.configuration.output }
```

Comment:
Did you accidentally omit the `id` and `conditionId` properties here? These are needed for the styles to work even if they were not necessary for the ConditionSet output to display.

Reply:
@shefalijoshi , yup. Sorry about that. I can add them back. I guess this is used in a different branch. grep for conditionId came up empty.